lunar/distribution.yaml

```diff
@@ -557,6 +557,17 @@ repositories:
       url: https://github.com/ros-visualization/qt_gui_core.git
       version: kinetic-devel
     status: maintained
+  random_numbers:
```

Comment:
Added in https://github.com/ros/rosdistro/pull/14332/commits/ba9e010c0c8974177cd1a1b3bd9237bedd0e00d2.